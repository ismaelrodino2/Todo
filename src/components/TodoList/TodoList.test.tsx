import { render } from '@testing-library/react';
import { TodoList } from '.';

describe('Tests for TodoList component', () => {
  it('Should match snapshot', async () => {
    const { container } = render(<TodoList />);

    expect(container).toMatchSnapshot();
  });
});
