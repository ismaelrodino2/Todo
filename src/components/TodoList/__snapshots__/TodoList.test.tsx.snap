// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for TodoList component Should match snapshot 1`] = `
<div>
  <ul>
    <div
      class="flex mb-4 flex-col w-full border border-gray-400 shadow-lg overflow-hidden m-auto pb-4"
    >
      <div
        class="justify-around	flex-col items-center flex"
      >
        <h1
          class=" font-montserrat underline  mt-2 ml-3"
        >
          To do list
        </h1>
      </div>
    </div>
  </ul>
</div>
`;
